### PR TITLE
Update export_sc_atlas_ebeye_xml.pl with solr credential

### PR DESCRIPTION
**Pull request by anilthanki**
Update export_sc_atlas_ebeye_xml.pl with solr credential  from environment

### CODE REVIEW

File: bin/export_sc_atlas_ebeye_xml.pl [hunk 1]
```diff
@@ -124,7 +125,12 @@ sub check_env_var {
 
 add_entry_count ( $configHash );
 
+sub LWP::UserAgent::get_basic_credentials {
+     my ($self, $realm, $url, $isproxy) = @_;
 
+     return $ENV{'SOLR_USER'}, $ENV{'SOLR_PASS'};
+}
```
Comment (pmb59) [inline bin/export_sc_atlas_ebeye_xml.pl:132]: Would it benefit wrapping this method [to turn off the redefine warning](https://perlmaven.com/lwp-useragent-and-basic-authentication)?
```
{
    no warnings 'redefine';
...
}

**Review by pmb59 — COMMENTED** [review]
@anilthanki LGTM

Can you confirm if this option to Access the Solr server using perl is working in your tests?

If any issue, as an alternative I see there's a specific module to interface with Solr
https://metacpan.org/pod/WebService::Solr

File: bin/export_sc_atlas_ebeye_xml.pl (hunk 1 above)
Comment (anilthanki) [reply bin/export_sc_atlas_ebeye_xml.pl:132]: It may be worth keeping the warning up for future generations..

**Comment by anilthanki** [comment]
> @anilthanki LGTM
> 
> Can you confirm if this option to Access the Solr server using perl is working in your tests?
> 
> If any issue, as an alternative I see there's a specific module to interface with Solr https://metacpan.org/pod/WebService::Solr

This run was performed using solr env vars http://gene-expression.ebi.ac.uk/jenkins/job/C1_SC_EBEYE_Export/29/console

**Review by pmb59 — APPROVED** [review]
LGTM

**Pull request merged**